src/test/java/uk/gov/hmcts/reform/ccd/document/am/controller/WelcomeTest.java

```diff
@@ -17,7 +20,22 @@ public void shouldReturnWelcomeMessage() {
         assertNotNull(caseDocumentControllerResponse, "No Response from WelcomeController");
         assertEquals(HttpStatus.OK, caseDocumentControllerResponse.getStatusCode(), "Status code is NOT OK");
         assertEquals(caseDocumentControllerResponse.getBody(),
-            "Welcome to CCD Case Document AM Controller",
-            "Response body does not have expected value");
+                     "Welcome to CCD Case Document AM Controller",
+                     "Response body does not have expected value");
+    }
+
+    @Test
+    public void shouldGetReturnIndex() {
+        String response = caseDocumentAmController.index();
+        assertNotNull(response, "Valid Response from WelcomeController");
+        assertEquals(response, "redirect:swagger-ui.html", "Response Value");
     }
+
+    @Test
+    public void shouldGetReturnIndex1() {
```

Comment:
Methode name should be more specific if possible

Reply:
Done